doc/source/users_guide/using-clm-tools/building-the-clm-tools.rst

```diff
@@ -99,7 +99,7 @@ The *README.filecopies* (which can be found in ``$CTSMROOT/tools``) is repeated
  Building the CLM tools that use the CIME configure/build system
 ================================================================
 
-**cprnc** and *gen_domain** both use the CIME configure/build system rather than the CLM specific version described above.
+``cprnc`` and ``gen_domain`` both use the CIME configure/build system rather than the CLM specific version described above.
 
-See `CIME documentation on adding grids <http://esmci.github.io/cime/users_guide/grids.html?highlight=gen_domain#adding-grids>`_ for more information on adding grids, creating mapping files, and running **gen_domain**. Also see the CIME file: ``$CTSMROOT/tools/mapping/gen_domain_files/INSTALL`` for how to build **gen_domain**.
+See `CIME documentation on adding grids <http://esmci.github.io/cime/users_guide/grids.html?highlight=gen_domain#adding-grids>`_ for more information on adding grids, creating mapping files, and running ``gen_domain``. Also see the CIME file: ``$CTSMROOT/tools/mapping/gen_domain_files/INSTALL`` for how to build ``gen_domain``.
```

Comment:
Maybe another todo, as domain files are not needed in NUOPC

Reply:
TODOs added in 6ac23b2.